### PR TITLE
Editorial cleanup

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes [AB#1756919](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1756919) - Editorial clean up of about topic and a few minor corrections.

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [x] Version 7.1 preview content
- [x] Version 7.0 content
- [x] Version 6 content
- [x] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/6/Microsoft.PowerShell.Core/About/about_Updatable_Help.md [hunk 1]
```diff
@@ -274,49 +276,48 @@ Save-Help -Module $m -DestinationPath C:\SavedHelp
...
 
 ```powershell
 Update-Help -Module DhcpServer -SourcePath E:\UsbFlashDrive\SavedHelp
 ```
 
-Without parameters, Save-Help downloads help for all modules in the session
+Without parameters, `Save-Help` downloads help for all modules in the session
 and for all installed modules that support Updatable Help. To be included,
 modules must be installed in directories that are listed in the value of the
 PSModulePath environment variable, on either the local computer or on a remote
```
Comment (chasewilson) [inline reference/6/Microsoft.PowerShell.Core/About/about_Updatable_Help.md:306]:
```suggestion
`$env:PSModulePath` environment variable, on either the local computer or on a remote
```

File: reference/6/Microsoft.PowerShell.Core/About/about_Updatable_Help.md [hunk 2]
```diff
@@ -274,49 +276,48 @@ Save-Help -Module $m -DestinationPath C:\SavedHelp
...
 computer for which you want to save help. These are also modules that are
 returned by running a "Get-Help -ListAvailable" command.
```
Comment (chasewilson) [inline reference/6/Microsoft.PowerShell.Core/About/about_Updatable_Help.md:308]:
```suggestion
returned by running a `Get-Help -ListAvailable` command.
```

File: reference/7.0/Microsoft.PowerShell.Core/About/about_Updatable_Help.md [hunk 3]
```diff
@@ -274,49 +276,48 @@ Save-Help -Module $m -DestinationPath C:\SavedHelp
...
 Update-Help -Module DhcpServer -SourcePath E:\UsbFlashDrive\SavedHelp
 ```
 
-Without parameters, Save-Help downloads help for all modules in the session
+Without parameters, `Save-Help` downloads help for all modules in the session
 and for all installed modules that support Updatable Help. To be included,
 modules must be installed in directories that are listed in the value of the
 PSModulePath environment variable, on either the local computer or on a remote
 computer for which you want to save help. These are also modules that are
 returned by running a "Get-Help -ListAvailable" command.
```
Comment (chasewilson) [inline reference/7.0/Microsoft.PowerShell.Core/About/about_Updatable_Help.md:308]:
```suggestion
returned by running a `Get-Help -ListAvailable` command.
```

File: reference/7.0/Microsoft.PowerShell.Core/About/about_Updatable_Help.md [hunk 4]
```diff
@@ -274,49 +276,48 @@ Save-Help -Module $m -DestinationPath C:\SavedHelp
...
 
 ```powershell
 Update-Help -Module DhcpServer -SourcePath E:\UsbFlashDrive\SavedHelp
 ```
 
-Without parameters, Save-Help downloads help for all modules in the session
+Without parameters, `Save-Help` downloads help for all modules in the session
 and for all installed modules that support Updatable Help. To be included,
 modules must be installed in directories that are listed in the value of the
 PSModulePath environment variable, on either the local computer or on a remote
```
Comment (chasewilson) [inline reference/7.0/Microsoft.PowerShell.Core/About/about_Updatable_Help.md:306]:
```suggestion
`$env:PSModulePath` environment variable, on either the local computer or on a remote
```

File: reference/7.1/Microsoft.PowerShell.Core/About/about_Updatable_Help.md [hunk 5]
```diff
@@ -274,49 +276,48 @@ Save-Help -Module $m -DestinationPath C:\SavedHelp
...
 Update-Help -Module DhcpServer -SourcePath E:\UsbFlashDrive\SavedHelp
 ```
 
-Without parameters, Save-Help downloads help for all modules in the session
+Without parameters, `Save-Help` downloads help for all modules in the session
 and for all installed modules that support Updatable Help. To be included,
 modules must be installed in directories that are listed in the value of the
 PSModulePath environment variable, on either the local computer or on a remote
 computer for which you want to save help. These are also modules that are
 returned by running a "Get-Help -ListAvailable" command.
```
Comment (chasewilson) [inline reference/7.1/Microsoft.PowerShell.Core/About/about_Updatable_Help.md:308]:
```suggestion
returned by running a `Get-Help -ListAvailable` command.
```

File: reference/7.1/Microsoft.PowerShell.Core/About/about_Updatable_Help.md [hunk 6]
```diff
@@ -274,49 +276,48 @@ Save-Help -Module $m -DestinationPath C:\SavedHelp
...
 
 ```powershell
 Update-Help -Module DhcpServer -SourcePath E:\UsbFlashDrive\SavedHelp
 ```
 
-Without parameters, Save-Help downloads help for all modules in the session
+Without parameters, `Save-Help` downloads help for all modules in the session
 and for all installed modules that support Updatable Help. To be included,
 modules must be installed in directories that are listed in the value of the
 PSModulePath environment variable, on either the local computer or on a remote
```
Comment (chasewilson) [inline reference/7.1/Microsoft.PowerShell.Core/About/about_Updatable_Help.md:306]:
```suggestion
`$env:PSModulePath` environment variable, on either the local computer or on a remote
```

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Updatable_Help.md [hunk 7]
```diff
@@ -138,48 +139,49 @@ Set-ExecutionPolicy cmdlet, type:
...
-To update help for the modules in the directory `$pshome\Modules`, you must be
+To update help for the modules in the directory `$pshome/Modules`, you must be
 member of the Administrators group on the computer.
 
-If you are not a member of the Administrators group, you cannot update help
-for these modules; but if you have Internet access, you can view help online
-in the TechNet Library.
+If you are not a member of the Administrators group, you cannot update help for
+these modules; but if you have internet access, you can view help online in the
+TechNet Library.
```
[the remark this answers is not in the document]
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Updatable_Help.md]: Fixed

**Pull request merged**